default/buildings.txt

```diff
@@ -1575,6 +1622,8 @@ BuildingType
         Population low = [[MIN_RECOLONIZING_SIZE]]
         OwnedBy empire = Source.Owner
     ]
+    EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
```

Comment:
I read something about restricting the number of colony/outpost bases per system from the roadmap. I dig out a little today, and may be the solution is to define enqueuelocation in ship_part.txt for CO_COLONY_POD and CO_OUTPOST_POD, similar to 

enqueuelocation = And [
    OwnedBy empire = Source.Owner
   [[SHIPDESIGN_HAS_NO_ENGINE]]
   [[SYSTEM_NOT_FULLY_COLONIZED]]
]

where 

SHIPDESIGN_HAS_NO_ENGINE
'''Or [
        DesignHasHull name = "SH_COLONY_BASE"
	// May be better use shiphull.StarlaneSpeed
	// Or the ship design has no engine
        And [
            Not DesignHasPart Low = 1 High=999 name = "FU_IMPROVED_ENGINE_COUPLINGS"
            Not DesignHasPart Low = 1 High=999 name = "FU_N_DIMENSIONAL_ENGINE_MATRIX"
            Not DesignHasPart Low = 1 High=999 name = "FU_SINGULARITY_ENGINE_CORE"
            Not DesignHasPart Low = 1 High=999 name = "FU_TRANSPATIAL_DRIVE"
        ]
    ] '''

SYSTEM_NOT_FULLY_COLONIZED
'''    Number low = 1 high=999 condition = And [
                Planet
                InSystem id = Source.SystemID
                Unowned
            ]'''

Unfortunately, I have no idea whether that would work (stuck again with "../libfreeorioncommon.so: undefined reference to `FreeOrionVersionString()'" when I tried to compile through kdevelop).